Mathlib/Analysis/SpecialFunctions/Gamma/Beta.lean

```diff
@@ -458,10 +458,95 @@ theorem Gamma_ne_zero_of_re_pos {s : ℂ} (hs : 0 < re s) : Gamma s ≠ 0 := by
   contrapose! hs
   simpa only [hs, neg_re, ← ofReal_natCast, ofReal_re, neg_nonpos] using Nat.cast_nonneg _
 
+lemma integral_re {X : Type*} [MeasurableSpace X]
+    (μ : Measure X) {f : X → ℂ} (hf : Integrable f μ) :
+    ∫ x, (f x).re ∂μ = (∫x, f x ∂μ).re := _root_.integral_re hf
+
+lemma setIntegral_re {X : Type*} [MeasurableSpace X]
+    (μ : Measure X) {s : Set X} {f : X → ℂ} (hf : IntegrableOn f s μ) :
+    ∫ x in s, (f x).re ∂μ = (∫x in s, f x ∂μ).re := Complex.integral_re _ hf.integrable
+
 end Complex
 
 namespace Real
 
+/-- The `Γ` function (of a real variable `s`). -/
+@[pp_nodot]
+def Beta (a b : ℝ) : ℝ :=
+  (Complex.betaIntegral a b).re
+
+lemma integrable_iff_ofReal {X : Type*} [MeasurableSpace X] {μ : Measure X}
+    {f : X → ℝ} : Integrable f μ ↔ Integrable (fun x ↦ (f x : ℂ)) μ :=
+    ⟨fun hf ↦ hf.ofReal, fun hf ↦ hf.re⟩
+
+lemma integrableOn_iff_ofReal {X : Type*} [MeasurableSpace X] {μ : Measure X} {s : Set X}
+    {f : X → ℝ} : IntegrableOn f s μ ↔ IntegrableOn (fun x ↦ (f x : ℂ)) s μ := integrable_iff_ofReal
```

Comment:
Please open a fourth PR for these, again it is independent from the rest. I'd put the first in `Mathlib.MeasureTheory.Function.L1Space`, next to `Integrable.ofReal`, the second in `Mathlib.MeasureTheory.Integral.IntegrableOn`, and add an `IntervalIntegrable` version please in `Mathlib.MeasureTheory.Integral.IntervalIntegral`.

Reply:
In fact you can put these results and the ones about `integral_re` in the same PR.